Firmware/menu.cpp

```diff
@@ -448,13 +426,9 @@ static void menu_draw_P(char chr, const char* str, int16_t val);
 template<>
 void menu_draw_P<int16_t*>(char chr, const char* str, int16_t val)
 {
-	int text_len = strlen_P(str);
-	if (text_len > 15) text_len = 15;
-	char spaces[LCD_WIDTH + 1] = {0};
-    memset(spaces,' ', LCD_WIDTH);
-	if (val <= -100) spaces[15 - text_len - 1] = 0;
-	else spaces[15 - text_len] = 0;
-	lcd_printf_P(menu_fmt_int3, chr, str, spaces, val);
+    char spaces[LCD_WIDTH + 1] = {0};
+    sprintf_P(spaces, menu_fmt_int3, chr, str, spaces, val);
+    lcd_print_pad_P(spaces, LCD_WIDTH);
```

Comment:
this one is not ok. Go for example to "Settings->Temperature" and you will see how it fails / crashes the firmware.

Reply:
Woops I see `spaces` appears twice there. Need to think about this one 🤔

Reply:
I reverted these changes for now.